src/Monitor/Monitor/Alerts/NewAzureRmMetricAlertRuleV2CriteriaCommand.cs

```diff
@@ -61,78 +74,85 @@ public class NewAzureRmMetricAlertRuleV2CriteriaCommand : MonitorCmdletBase
         /// Gets or sets the rule condition operator
         /// </summary>
         [Parameter(Mandatory = true, HelpMessage = "The rule condition operator")]
-        [PSArgumentCompleter("GreaterThan", "GreaterThanOrEqual", "LessThan", "LessThanOrEqual")]
+        [PSArgumentCompleterWithScript(OperatorCompleterScript)]
         public String Operator { get; set; }
 
         /// <summary>
         /// Gets or sets the rule threshold
         /// </summary>
         [Parameter(ParameterSetName = StaticThresholdParameterSet, Mandatory = true,  HelpMessage = "The threshold for rule condition")]
         public double Threshold { get; set; }
-
-        /// <summary>
-        /// Gets or sets the rule DynamicThreshold
-        /// </summary>
-        [Parameter(ParameterSetName = DynamicThresholdParameterSet, Mandatory = true, HelpMessage = "The Dynamic Threshold for rule condition")]
-        public string DynamicThreshold { get; set; }
-
+        
         /// <summary>
         /// Gets or sets the rule sensitivity
         /// </summary>
         [Parameter(ParameterSetName = DynamicThresholdParameterSet, Mandatory = false, HelpMessage = "The sensitivity for rule condition")]
         [PSArgumentCompleter("Low", "Medium", "High")]
-        public String Sensitivity { get; set; }
+        public String ThresholdSensitivity { get; set; } = "Medium";
```

Comment:
Breaking Change: Sensitivity -> ThresholdSensitivity: renaming parameter is only allowed if you create a parameter alias to the original parameter name